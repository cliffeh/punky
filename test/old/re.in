re/(abc)/                  ; re/(abc)/
re/(abc)/e                 ; re/(abc)/e
re/(abc)/ie                ; re/(abc)/ie
(re/a(b)c/ "abc")          ; ()
(re/a(b)c/e "abc")         ; ("abc" "b")
(re/a(b)c/ei "AbC")        ; ("AbC" "b")
(re/(f(o)*bar)/e "foobar") ; ("foobar" "foobar" "o")
(re/(f(o*)bar)/e "foobar") ; ("foobar" "foobar" "oo")
