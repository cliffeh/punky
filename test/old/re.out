re/(abc)/
re/(abc)/e
re/(abc)/ie
()
("abc" "b")
("AbC" "b")
("foobar" "foobar" "o")
("foobar" "foobar" "oo")
